azure-pipeline-templates/build.yml

```diff
@@ -187,10 +187,10 @@ steps:
     retryCountOnTaskFailure: 3
 
   # azcli login
-  - script: |
-      az login --identity --username $(ACLI_BLOBFUSE_MSI_APP_ID)
-    displayName: 'Azure CLI login'
-    condition: eq('${{ parameters.skip_azcli }}', 'false')
+  # - script: |
```

Comment:
MSI login using username field is depricated and if I migrate this step to work with client-id it fails with 400 error. Tried this step manually on vm as well but it consistently fails to login. Hence for now we are skipping this part and cli login based test.